packages/mobile-native-mcp-server/src/workflow/metadata.ts

```diff
@@ -52,6 +52,7 @@ export type WorkflowUserInputProperties = typeof WORKFLOW_USER_INPUT_PROPERTIES;
  * Defines the structure of state that flows through the workflow nodes
  */
 export const MobileNativeWorkflowState = Annotation.Root({
+  projectDirectory: Annotation<string>,
```

Comment:
I think this may have snuck in from some of our early work. I assume you're not using it? I don't see it referenced in the project anywhere else.